docs/features.md

```diff
@@ -542,6 +542,53 @@ variations:
         value: '{"title": "Welcome to our website", "subtitle": "We are glad you are here"}'
 ```
 
+## Parents
```

Comment:
I think the "parent-children" relationship is easier to grasp than "required" so I liked it more. It just gets weaker when you start adding the multi-parent feature below. That's not super intuitive since for me there's only one parent (I know I'm not helping, just sharing the my thoughts)

Reply:
very much the same situation with my thinking as well.

I originally started with `parent` to require a certain other feature to be enabled for this current feature to be evaluated as enabled.

then I realized, it makes sense to have more than one required features, and then it became `parents`.

and now it doesn't make sense because you shouldn't have multiple parents.

leaning a bit more towards `required` currently myself for this reason.

Reply:
either way, there is no option that I consider wrong.